lib/forge_sdk/protocol/display/datetime.ex

```diff
@@ -6,7 +6,11 @@ defimpl ForgeSdk.Display, for: Google.Protobuf.Timestamp do
   alias ForgeSdk.DateTimeUtils
 
   def display(%{seconds: seconds}, expand? \\ false) do
-    dt = DateTime.from_unix!(seconds)
+    dt =
+      case DateTime.from_unix(seconds) do
+        {:ok, date} -> date
+        _ -> DateTime.from_unix!(0)
```

Comment:
I don't think we shall modify this part like this. Instead, we shall enable ForgeWeb to handle exception like this.
Displaying zero will cause unexpected error.